glean-core/rlb/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "glean-preview"
+name = "glean"
 version = "0.0.5"
 authors = ["Jan-Erik Rediger <jrediger@mozilla.com>", "The Glean Team <glean-team@mozilla.com>"]
-description = "Nice Glean SDK Rust API"
+description = "Glean SDK Rust language bindings"
```

Comment:
I can't comment on a moved-only file, but the glean-core/rlb/README.md needs changes too

Reply:
Done!